### PR TITLE
Added controls to run OTA for Device Client Auth component

### PR DESCRIPTION
**Pull request by esmirnov-klika-tech**
**Issue #, if available:** GGMQ-33 

**Description of changes:** Modified uat directory structure and added multiproject config

**Why is this change necessary:** To have ability to use OTA framework

**How was this change tested:** Manually

**Any additional information or context required to review the change:**

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/uat.yml [hunk 1]
```diff
@@ -26,8 +26,7 @@ jobs:
         with:
           java-version: 1.8
       - name: Build with Maven (not Windows)
-        working-directory: uat
-        run: mvn -ntp -U clean verify
+        run: mvn -DskipTests=true -ntp -U clean verify & mvn -U -ntp verify -f uat/pom.xml
```
Comment (bgklika) [inline .github/workflows/uat.yml:29]: 1. Seems first command rebuild  main project of Auth plugin - that does in other CI workflow, see maven.yml 

2. & run first comand in the shell background.

File: uat/custom-components/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Test MQTT5 client bases on AWS IoT Device SDK for Java v2
```
Comment (bgklika) [inline uat/custom-components/README.md:1]: We going to have several MQTT clients. 
Does it mean we will move SDK-based client one more time ?

File: uat/testing-features/src/main/resources/greengrass/features/client-devices-auth-1.feature [hunk 3]
```diff
@@ -0,0 +1,10 @@
+@Auth
+Feature: Client Devices Auth
+
+    Scenario: Auth-1-T1-a: Registration and Run
+        Given my device is registered as a Thing
+        And my device is running Greengrass
+        When I create a Greengrass deployment with components
+            | aws.greengrass.clientdevices.Auth | LATEST |
+        And I deploy the Greengrass deployment configuration
+        Then the Greengrass deployment is COMPLETED on the device after 300 seconds
```
Comment (bgklika) [inline uat/testing-features/src/main/resources/greengrass/features/client-devices-auth-1.feature:10]: Please add \n after last line.

File: uat/pom.xml [hunk 4]
```diff
@@ -290,74 +78,6 @@
...
-    </properties>
-    <distributionManagement>
-        <snapshotRepository>
-            <id>greengrass-dev-snapshot</id>
-            <name>Greengrass Dev Snapshot</name>
-            <url>${dev-snapshot-repository-url}</url>
-        </snapshotRepository>
-    </distributionManagement>
-</project>
+</project>
```
Comment (bgklika) [inline uat/pom.xml:83]: Missing end of line (\n) in last line of text file.

File: uat/testing-features/src/main/resources/greengrass/features/client-devices-auth-1.feature (hunk 3 above)
Comment (esmirnov-klika-tech) [reply uat/testing-features/src/main/resources/greengrass/features/client-devices-auth-1.feature:10]: fixed

File: .github/workflows/uat.yml (hunk 1 above)
Comment (esmirnov-klika-tech) [reply .github/workflows/uat.yml]: fixed 736260a31d58a2c3a4a796482a041b170d3c1d37

File: uat/custom-components/README.md (hunk 2 above)
Comment (esmirnov-klika-tech) [reply uat/custom-components/README.md]: Project structure was updated according to discussed solution ba76450deb76f111510165f55bac5a57e3d60a8e

**Review by bgklika — APPROVED** [review]

**Pull request closed without merging**